openedx/core/djangoapps/credit/tests/test_views.py

```diff
@@ -365,40 +367,18 @@ def _serialize_credit_course(self, credit_course):
             'enabled': credit_course.enabled
         }
 
-    def test_session_auth(self):
```

Comment:
would it make sense to leave this test in place, and modifying it to ensure that the calls return 403 regardless of whether the user is staff or not?